apps/google-analytics-4/frontend/src/components/main-app/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import tokens from '@contentful/f36-tokens';
+
+const ACCESSIBILITY_LABEL = 'Analytics line chart';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip);
+
+ChartJS.defaults.font.size = 12;
+ChartJS.defaults.font.family = tokens.fontStackPrimary;
+ChartJS.defaults.font.weight = '500';
+ChartJS.defaults.borderColor = tokens.gray200;
+ChartJS.defaults.datasets.line.borderColor = tokens.colorPrimary;
+
+interface LineChartProps {
+  dataValues: Array<string | number>;
```

Comment:
Nit: TS best practices usually have array values as `string[]` instead of the `Array<string>` class.

Reply:
Oh yes good point, will adjust!